src/en/config-azure.md

```diff
@@ -1,123 +1,223 @@
-# Configuring for Windows Azure
+Title: Juju Azure provider
+TODO: Decide on what to do with provider-specific features (e.g. placement).
 
-This process requires you to have a Windows Azure account. If you have not
-signed up for one yet, it can obtained at http://azure.microsoft.com/en-us/.
 
-You should start by generating a generic configuration file for Juju, using the
-command:
+# Overview
+
+Juju now supports Microsoft Azure's Resource Manager API. The Azure provider
```

Comment:
I sort of feel that this whole paragraphe should be a note or something - this is information for existing Azure users